File: src/main/java/org/taktik/icure/dto/filter/code/CodeByRegionTypeLabelLanguageFilter.java
package org.taktik.icure.dto.filter.code;

import org.taktik.icure.dto.filter.Filter;
import org.taktik.icure.entities.base.Code;

public interface CodeByRegionTypeLabelLanguageFilter extends Filter<String, Code> {
	public String getRegion();
	public String getType();
	public String getLanguage();
	public String getLabel();
}

File: src/main/kotlin/org/taktik/icure/domain/filter/code/CodeByRegionTypeLabelLanguageFilter.kt
package org.taktik.icure.domain.filter.code

import org.taktik.icure.domain.filter.Filter
import org.taktik.icure.entities.base.Code

interface CodeByRegionTypeLabelLanguageFilter : Filter<String, Code> {
    val region: String?
    val type: String?
    val language: String?
    val label: String?
}
